website/source/docs/telemetry/metrics.html.md

```diff
@@ -0,0 +1,805 @@
+    <td>Total CPU resources consumed by the task in the system space</td>
+    <td>Percentage</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.cpu.user`</td>
+    <td>Total CPU resources consumed by the task in the user space</td>
+    <td>Percentage</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.cpu.throttled_time`</td>
+    <td>Total time that the task was throttled</td>
+    <td>Nanoseconds</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.cpu.total_ticks`</td>
+    <td>CPU ticks consumed by the process in the last collection interval</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+  </tr>
+</table>
+
+## Job Metrics
+
+Job metrics are emitted by the Nomad leader server.
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th>Metric</th>
+    <th>Description</th>
+    <th>Unit</th>
+    <th>Type</th>
+    <th>Labels</th>
+  </tr>
+  <tr>
+    <td>`nomad.job_summary.queued`</td>
+    <td>Number of queued allocations for a job</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+    <td>job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.job_summary.complete`</td>
+    <td>Number of complete allocations for a job</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+    <td>job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.job_summary.failed`</td>
+    <td>Number of failed allocations for a job</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+    <td>job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.job_summary.running`</td>
+    <td>Number of running allocations for a job</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+    <td>job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.job_summary.starting`</td>
+    <td>Number of starting allocations for a job</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+    <td>job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.job_summary.lost`</td>
+    <td>Number of lost allocations for a job</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+    <td>job, task_group</td>
+  </tr>
+</table>
+
+## Metric Types
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th>Type</th>
+    <th>Description</th>
+    <th>Quantiles</th>
+  </tr>
+  <tr>
+    <td>Gauge</td>
+    <td>
+        Gauge types report an absolute number at the end of the aggregation
+        interval
+    </td>
+    <td>false</td>
+  </tr>
+  <tr>
+    <td>Counter</td>
+    <td>
+        Counts are incremented and flushed at the end of the aggregation
+        interval and then are reset to zero
+    </td>
+    <td>true</td>
+  </tr>
+  <tr>
+    <td>Timer</td>
+    <td>
+        Timers measure the time to complete a task and will include quantiles,
+        means, standard deviation, etc per interval.
+    </td>
+    <td>true</td>
+  </tr>
+</table>
+
+## Tagged Metrics
```

Comment:
as of https://github.com/hashicorp/nomad/pull/5847, there are also labels for namespace. not sure whether that's worth adding somewhere on this page or not.

Reply:
Maybe just add the line at then end?

`Nomad supports labels for namespaces as well.`